web-jehlomat/src/screens/Nalezy/Components/FilterByState.tsx

```diff
@@ -1,26 +1,42 @@
-import React, { FunctionComponent, useEffect, useState } from 'react';
+import React, { FunctionComponent, useCallback, useEffect, useState } from 'react';
+import { useSetRecoilState } from 'recoil';
 import { SyringeState } from '../types/SyringeState';
 import { Filter } from './Filter';
 import Select from './Select';
+import { filteringState } from 'screens/Nalezy/store';
+import { Filtering } from 'screens/Nalezy/types/Filtering';
 
-interface FilterStateProps {
-    onFilter: (state: SyringeState) => void;
-    onReset: () => void;
-}
-
-const FilterByState: FunctionComponent<FilterStateProps> = ({ onFilter, onReset }) => {
+const FilterByState: FunctionComponent = () => {
     const [state, setState] = useState<SyringeState | ''>('');
```

Comment:
not your code, but could we use null instead of empty string here?

Reply:
Same as before, be able to reset select.